101-kusto-cluster-database/azuredeploy.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "Clusters_kustocluster_name": {
+          "type": "string",
+          "metadata": {
```

Comment:
add a defaultValue - globally unique if needed

Reply:
Per the info here: https://github.com/Azure/azure-quickstart-templates/wiki/defaultValue-vs.-azuredeploy.parameters.json-vs.-GEN-UNIQUE I see it's recommended to add a GEN-UNIQUE in the properties and remove the defaultValue in azuredeploy.json therefore I did that. Let me know if I should correct anything.

Reply:
ah, ok... the page is a bit dated (I updated it)...  a defaultValue can contain an expression that is globally unique, this makes it easier for users to deploy in a "kick the tires" mode (and prod if nec).  You can use:

"defaultValue": "[concat('kusto-', uniqueString(resourceGroup().id))]"

provided a '-' is allowed in the name, if not just remove it...